openfl/component/aggregator/straggler_handling.py

```diff
@@ -2,27 +2,77 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-"""Cutoff time based Straggler Handling function."""
+"""Straggler handling module."""
+
 import threading
 import time
+from abc import ABC, abstractmethod
 from logging import getLogger
 from typing import Callable
 
 import numpy as np
 
-from openfl.component.straggler_handling_functions.straggler_handling_function import (
-    StragglerHandlingPolicy,
-)
 
+class StragglerHandlingPolicy(ABC):
+    """Federated Learning straggler handling interface."""
+
+    @abstractmethod
+    def start_policy(self, **kwargs) -> None:
+        """
+        Start straggler handling policy for collaborator for a particular round.
+        NOTE: Refer CutoffPolicy for reference.
```

Comment:
Why is `CutoffPolicy` mentioned here?

Reply:
To showcase the `CutoffPolicy's` `start_policy` implementation as a reference for users.